src/main/java/org/opensearch/flowframework/client/MLClient.java

```diff
@@ -22,12 +22,12 @@ private MLClient() {}
     /**
      * Creates machine learning client.
      *
-     * @param nodeClient node client of OpenSearch.
+     * @param client client of OpenSearch.
      * @return machine learning client from ml-commons.
      */
-    public static MachineLearningNodeClient createMLClient(NodeClient nodeClient) {
+    public static MachineLearningNodeClient createMLClient(Client client) {
```

Comment:
what's the motivation of this client change?

Reply:
MLClient should take care of making the client as NodeClient
```java
@Generated
    public MachineLearningNodeClient(Client client) {
        this.client = client;
    }